reference/readonly-sdks/ruby/OpenAPI.json

```diff
@@ -186,6 +186,58 @@
         "type": "apiKey",
         "name": "X-Management-Token",
         "in": "header"
+      },
+      "X-Voucherify-OAuth": {
```

Comment:
I wonder about those changes. It seems that You merged two PRs. One with C# changes. One from oAuth. For me, it could go together. But for backward compatibility's sake, we should consider dividing those changes.